recipe-hooks-project/src/AppPage.test.js

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import {cleanup, fireEvent, render} from "@testing-library/react";
+import {BrowserRouter} from "react-router-dom";
+import axios from "axios";
+import {AuthContext} from "./contexts/AuthenticationProvider";
+import AppPage from "./AppPage";
+import Login from "./Login";
+import {waitForElement} from "@testing-library/dom";
+
+jest.mock("axios");
+
+afterEach(() => {
+  cleanup()
```

Comment:
You don't need to do this anymore, testing-library automatically invokes `cleanup` after each test now

Reply:
I didn't know that!